insights/parsers/awx_manage.py

```diff
@@ -7,6 +7,8 @@
 AnsibleTowerLicenseType - command ``/usr/bin/awx-manage check_license``
 
 AnsibleTowerLicense - command ``/usr/bin/awx-manage check_license --data``
+
+AwxManagePrintSettings - command ``awx-manage print_settings``
 --------------------------------------------------------------------------
```

Comment:
@wushiqinlou I believe this is causing a formatting issues even though it doesn't throw errors.  I believe you can just removed it.  Also on lines 33-36 below, the examples should be indented 4 spaces to format correctly.

Reply:
Updated, please have a check.